primitives/xcm/src/lib.rs

```diff
@@ -49,3 +58,66 @@ impl<
 		Ok((what, amount))
 	}
 }
+
+pub struct TrappistDropAssets<
```

Comment:
TrappistAssetTrap might be a slightly better name, mostly as I associate drop with a database drop statement so find it a little confusing (i.e. destroying assets)

Reply:
The rationale behind this name is to signal that this is Trappist's implementation of the `DropAssets` trait.

I think the term drop comes from "drop Holding Register".

Conceptually speaking, the Asset Trap is composed of both `DropAssets` and `ClaimAssets` traits. So `TrappistAssetTrap` wouldn't encapsulate that meaning.

Our `XcmConfig` is using `TrappistDropAssets` as the implementation for `DropAssets`, but `pallet-xcm` for `ClaimAssets`.

Reply:
Makes sense now, thanks!